src/oatpp/network/server/Server.hpp

```diff
@@ -103,8 +114,9 @@ class Server : public base::Countable {
   /**
    * Call &id:oatpp::network::ConnectionProvider::getConnection; in the loop and passes obtained Connection
    * to &id:oatpp::network::server::ConnectionHandler;.
+   * @param blocking - Start the server blocking (thread of callee) or non-blocking (own thread)
    */
-  void run();
+  void run(bool blocking = true);
```

Comment:
The name "blocking"/"non-blocking" is confusing to me in this context.
I think it's better to describe exactly what it's going to do:

```cpp
Server::run(bool createNewThread);
```

Reply:
See cpp